`src/lib/api.ts` 파일 누락로 인한 빌드 오류 (Module not found: '../../lib/api')

### Description
Github actions 동작 중 빌드 에러 발생
https://github.com/Pseudo-Lab/DevFactory/actions/runs/19400936845 참고

### Steps / How to Reproduce
1. 프론트엔드 디렉토리 (frontend)에서 `npm run build` 실행 또는 CI 파이프라인 실행  
2. `src/app/page2/page.tsx`, `src/app/page3/page.tsx`, `src/components/Header.tsx` 등에서 아래 import가 존재함:  
   ```ts
   import { authenticatedFetch } from '../../lib/api';
   ```  
3. 그러나 `frontend/src/lib/api.ts` 파일이 존재하지 않아 모듈을 찾지 못해 빌드 실패

### Expected vs Actual
- Expected: 프론트엔드 빌드가 정상 완료되고 이후 배포 단계(CD)가 이어져야 합니다.
- Actual: 다음 에러로 빌드가 중단되었습니다:
```Module not found: Can't resolve '../../lib/api'```

### Attachments
에러 로그 중 일부
```
#21 25.41 ./src/components/Header.tsx:3:1
#21 25.41 Module not found: Can't resolve '../lib/api'
#21 25.41   1 | 'use client';
#21 25.41   2 |
#21 25.41 > 3 | import { authenticatedFetch } from '../lib/api';
#21 25.41     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#21 25.41   4 | import React, { useState, useEffect } from 'react';
#21 25.41   5 | import { useFormStore } from '../store/formStore';
#21 25.41   6 |
```

--- a/getcloser/frontend/src/lib/api.ts
+++ b/getcloser/frontend/src/lib/api.ts
@@ -1,0 +1,18 @@
+import { useFormStore } from '../store/formStore';
+
+export async function authenticatedFetch(input: RequestInfo | URL, init?: RequestInit): Promise<Response> {
+  const { accessToken } = useFormStore.getState();
+
+  const headers = new Headers(init?.headers);
+
+  if (accessToken) {
+    headers.set('Authorization', `Bearer ${accessToken}`);
+  }
+
+  const newInit: RequestInit = {
+    ...init,
+    headers,
+  };
+
+  return fetch(input, newInit);
+}

